packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -20,6 +19,16 @@ import IntegrationProviderOAuth1 from './IntegrationProviderOAuth1'
 import {JiraServerIssueConnection} from './JiraServerIssue'
 import JiraServerRemoteProject from './JiraServerRemoteProject'
 import TeamMemberIntegrationAuthOAuth1 from './TeamMemberIntegrationAuthOAuth1'
+import IntegrationRepoId from "~/shared/gqlIds/IntegrationRepoId";
+
+type IssueArgs = {
+  first: number
+  after?: string
+  queryString: string | null
+  isJQL: boolean
+  projectKeyFilters: string[] | null
+  [argName: string]: any
```

Comment:
-1 why `argName: string`? When is this used?

Reply:
I don't see it used anywhere, so removed